parso/_compatibility.py

```diff
@@ -50,6 +50,12 @@ def u(string):
 except NameError:
     # Python 2.7 (both IOError + OSError)
     FileNotFoundError = EnvironmentError
+try:
+    # Python 2.7
+    PermissionError = PermissionError
+except NameError:
+    # Python 3.3+
```

Comment:
I find this comments a bit confusing, shouldn't this be `Python 2.7` and the upper one `Python 3.3+`?

Reply:
Yeah it's definitely wrong. No idea how I messed that up :)